src/index.html

```diff
@@ -15,12 +15,15 @@
         <script src="js/vendor/modernizr-2.8.3.min.js"></script>
     </head>
     <body>
-        <!--[if lt IE 8]>
-            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
+      <section class="container">
```

Comment:
What other semantic HTML can you use instead of `<sections>` within `<sections>`?